### PR TITLE
Fix typo in 2019 survey blog post

### PR DESCRIPTION
**Pull request by camelid**
the -> they

### CODE REVIEW

File: posts/2020-04-17-Rust-survey-2019.md
Comment (camelid) [inline posts/2020-04-17-Rust-survey-2019.md:36]: I would also suggest changing this to something like:

```suggestion
If we compare this to the 2018 survey – where 75% of responders indicated they used Rust, 8% indicated they did not currently use Rust but had used it in the past, and 8% indicated they had never used Rust – more responders were using Rust in 2019.
```

**Review by XAMPPRocky — APPROVED** [review]
Thank you for your PR!

**Comment by camelid** [comment]
@XAMPPRocky Do you want to merge this?

**Comment by jyn514** [comment]
@camelid it requires approval from core, not just the website owner.

(This is what I mentioned in https://github.com/rust-lang/blog.rust-lang.org/pull/670#issuecomment-674105146, which I still think should be changed.)

**Comment by camelid** [comment]
@jyn514 Ah, thanks for the clarification. Is there any way for me to ping the core team?

**Comment by Mark-Simulacrum** [comment]
Considering it was opened less than 24 hours ago, I would not rush with pinging folks - core is quite busy. That said, merging.

**Pull request merged**
